### ISSUE TITLE
Problem during sided_setup when used with Curios API

### ISSUE THREAD

**Issue by tiffer642** [Andrew6rant/Tiered#13]
I get this crash report https://pastebin.com/QGypXBBy whenever I use tiered with Curios API, it only happens with tiered enabled and if it's disabled it works fine. I've tried other versions of both mods and I'm not able to figure out what exactly causes it. It was working before a couple weeks ago but now it has stopped. this might be good to look into.

**Comment by killer2gudpl** [comment]
yup, same. https://github.com/TheIllusiveC4/Curios/issues/344 references the issue as the issue with Tiered

**Comment by Theriople** [comment]
ive got the same problem
